src/pages/Campaign/CampaignPage/components/Tabs/CharacterTab/CharacterCard/CharacterCard.tsx

```diff
@@ -146,36 +160,40 @@ export function CharacterCard(props: CharacterCardProps) {
             sx={{ mr: 1, mt: 1 }}
           />
         </Box>
-        <Accordion
-          variant={"outlined"}
-          sx={{ borderLeftWidth: 0, borderRightWidth: 0 }}
-        >
-          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-            Assets
-          </AccordionSummary>
-          <AccordionDetails>
-            <Stack spacing={2} px={2}>
-              {storedAssets?.map((storedAsset, index) => (
-                <AssetCard
-                  key={index}
-                  storedAsset={storedAsset}
-                  assetId={storedAsset.id}
-                />
-              ))}
-            </Stack>
-          </AccordionDetails>
-        </Accordion>
-        <Accordion
-          variant="outlined"
-          sx={{ borderLeftWidth: 0, borderRightWidth: 0 }}
-        >
-          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-            {trackLabel}
-          </AccordionSummary>
-          <AccordionDetails>
-            <TrackComponent characterId={characterId} />
-          </AccordionDetails>
-        </Accordion>
+        <List>
+          <ListItemButton
+              divider={true}
+              onClick={handleExpandAssets}
+              sx={(theme) => ({ borderTop: `1px solid ${theme.palette.divider}` })}
+          >
```

Comment:
If we go this route, we just need to provide some context to screen readers about which state things are in - expanded or collapsed. There's probably an `aria-` property for it.